docker/images/kuma_base/Dockerfile

```diff
@@ -1,54 +1,121 @@
-FROM quay.io/mozmar/base:latest
-
-ENV PYTHONDONTWRITEBYTECODE=1 \
-    PIP_DISABLE_PIP_VERSION_CHECK=1
+FROM python:2.7-slim
 
+# extra python env
+ENV PYTHONDONTWRITEBYTECODE=1
```

Comment:
It should be revoked in the kuma full image that goes to production. As we need bytecode in production.
you should add `PYTHONDONTWRITEBYTECODE=0` in `images/kuma/Dockerfile`

Reply:
Good catch, thanks again @safwanrahman. I'll change in another PR.

Reply:
I think @pmac had good reasons for ``PYTHONDONTWRITEBYTECODE=1``, but I can't find them.

Reply:
Reasons were 1) it looks to be common in python images because 2) the .pyc files are useful when an app starts the 2nd time. In our containers this rarely happens since each container start is a fresh system.

Reply:
I suspect it could slow down Docker for Mac as well, writing (Linux only?) ``.pyc`` files to the slow shared filesystem.